core/src/test/scala/akka/persistence/cassandra/query/EventsByTagStageSpec.scala

```diff
@@ -51,8 +51,10 @@ object EventsByTagStageSpec {
        |   first-time-bucket = "${today.minusMinutes(5).format(firstBucketFormat)}"
        |   max-result-size-query = $fetchSize
        |   log-queries = on
-       |   refresh-interval = 250ms
-       |   events-by-tag-gap-timeout = 2s
+       |   refresh-interval = 200ms
```

Comment:
should we move the refresh-interval inside events-by-tag section or wait with that until we do the big config refactoring?

Reply:
It's used for events by persistence id as well so maybe not this time but we could add an override in this section later

Reply:
added a note for later https://github.com/akka/akka-persistence-cassandra/issues/81#issuecomment-348537776